### DISCUSSION TITLE
Bullets not disappearing

### DISCUSSION THREAD

**Discussion by bshih** [TfTHacker/DashboardPlusPlus#17] [General]
I tried using the css and couldn't get the bullets to disappear for some reason despite a bunch of fiddling. So then I tried downloading the full vault, and the same thing seems to be happening, though everything else seems to work.
<img width="626" alt="Screenshot 2023-07-23 at 8 33 35 PM" src="https://github.com/TfTHacker/DashboardPlusPlus/assets/967511/45dae3f9-21ee-4d41-96b1-8c77435c5c23">

Any idea what might be going on?

**Comment by TfTHacker** [comment]
Hi. Do you mean the bullet next to the list item? If so, the bullet is supposed to be there. Unless you mean some where else?

**Reply by bshih** [reply]
Hi - I meant the bullet on the top level list (e.g., Sunday Game) - in the screenshots from here https://medium.com/obsidian-observer/dashboard-a-simple-organization-and-navigation-method-for-obsidian-vaults-2b1982d023a0 I thought that only the second level list items were supposed to have a bullet.

That's also what I thought the "list-style: none" was supposed to be doing, but I might be mistaken.

I also was curious why the vertical line shows up in my version of the demo vault where it doesn't in the screenshots.

**Comment by TfTHacker** [comment]
You are right. Some how I didn't notice this. I am not sure why. I have now posted an updated vault in this repository. Grab the newest dashboard.css.

Thank you for reporting this.

**Reply by bshih** [reply]
You are a champ, thank you for the fix. My obsidian-css-fu is clearly lacking. Cheers!
